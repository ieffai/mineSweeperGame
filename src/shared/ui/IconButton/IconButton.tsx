import { default as BaseIconButton } from '@mui/material/IconButton';
import type { IconButtonProps as BaseIconButtonProps } from '@mui/material/IconButton';

export type IconButtonProps = BaseIconButtonProps;
export const IconButton = BaseIconButton;
