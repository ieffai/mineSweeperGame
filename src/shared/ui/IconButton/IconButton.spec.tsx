import { render } from '@testing-library/react';
import { IconButton } from './IconButton';

describe('IconButton', () => {
  it('should render successfully', () => {
    const { asFragment } = render(<IconButton />);
    expect(asFragment()).toMatchSnapshot();
  });
});
