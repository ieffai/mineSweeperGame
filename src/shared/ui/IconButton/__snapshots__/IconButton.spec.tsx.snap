// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`IconButton > should render successfully 1`] = `
<DocumentFragment>
  <button
    class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
    tabindex="0"
    type="button"
  />
</DocumentFragment>
`;
